mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -183,12 +188,36 @@ const ChatTaskpane: React.FC<IChatTaskpaneProps> = ({
         }
     }
 
+    const rejectAICodeIfNeeded = (chatHistoryManager: ChatHistoryManager) => {
+        // If the user has sends a msg, and code diffs are displayed, 
+        // then automatically reject the AI code before sending a new message.
+        if (unifiedDiffLines !== undefined) {
+            const codeCellID = chatHistoryManager.getCodeCellIDOfMostRecentAIMessage() || ''
+            rejectAICode(codeCellID, false)
+        }
+    }
+
+    const getCodeCellIDWithDiff = (codeCellID: string): string => {
```

Comment:
Do we need this function? Don't we always have the cell id stored in the chat history manager? We don't need to handle it differently if that is currently the active code cell or if it's not the active code cell. I might be missing something though.